cmd/common/image/build.go

```diff
@@ -320,14 +320,29 @@ func buildInContainer(opts *cmdOpts, conf *buildFile) error {
 		return err
 	}
 
+	// where the gadget source code is mounted in the container
+	gadgetSourcePath := "/work"
+	pathHost := cwd
+
+	inspektorGadetSrcPath := os.Getenv("INSPEKTOR_GADGET_PATH")
```

Comment:
The commitmsg says `--inspektor-gadget-path`. I guess it needs to be updated.

Reply:
Could the env var name make it clear it is about building gadgets?

`$GADGET_BUILD_FROM_IG_SOURCE` or something. Naming is hard.
I am not sure if other variables `git grep os.Getenv` are named consistently.

Reply:
Does this variable also make use of the modified headers we have in include/gadget/? This is a common problem for me when I update the headers and compile the gadgets in-tree.

Reply:
> Could the env var name make it clear it is about building gadgets?

I used `IG_SOURCE_PATH`, is it better? (Btw, I'm not supper worried about this name as it's mostly for us, external users won't build in-tree gadgets)

> Does this variable also make use of the modified headers we have in include/gadget/? This is a common problem for me when I update the headers and compile the gadgets in-tree.

I added a line to handle this case as well, thanks for the suggestion!